{
	"id": "oval:com.redhat.rhsa:def:20220667",
	"version": "636",
	"class": "patch",
	"metadata": {
		"title": "RHSA-2022:0667: python-pillow security update (Important)",
		"reference": [
			{
				"ref_id": "RHSA-2022:0667",
				"ref_url": "https://access.redhat.com/errata/RHSA-2022:0667",
				"source": "RHSA"
			},
			{
				"ref_id": "CVE-2022-22816",
				"ref_url": "https://access.redhat.com/security/cve/CVE-2022-22816",
				"source": "CVE"
			},
			{
				"ref_id": "CVE-2022-22817",
				"ref_url": "https://access.redhat.com/security/cve/CVE-2022-22817",
				"source": "CVE"
			}
		],
		"description": "The python-pillow packages contain a Python image processing library that provides extensive file format support, an efficient internal representation, and powerful image-processing capabilities.\n\nSecurity Fix(es):\n\n* python-pillow: PIL.ImageMath.eval allows evaluation of arbitrary expressions (CVE-2022-22817)\n\n* python-pillow: buffer over-read during initialization of ImagePath.Path in path_getbbox() in path.c (CVE-2022-22816)\n\nFor more details about the security issue(s), including the impact, a CVSS score, acknowledgments, and other related information, refer to the CVE page(s) listed in the References section.",
		"advisory": {
			"from": "secalert@redhat.com",
			"severity": "Important",
			"updated": {
				"date": "2022-02-24"
			},
			"cve": [
				{
					"text": "CVE-2022-22816",
					"cvss3": "6.5/CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:L/A:L",
					"cwe": "CWE-125",
					"href": "https://access.redhat.com/security/cve/CVE-2022-22816",
					"impact": "moderate",
					"public": "20220102"
				},
				{
					"text": "CVE-2022-22817",
					"cvss3": "9.8/CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:H/A:H",
					"cwe": "CWE-77",
					"href": "https://access.redhat.com/security/cve/CVE-2022-22817",
					"impact": "important",
					"public": "20220102"
				}
			],
			"affected": {},
			"affected_cpe_list": {
				"cpe": [
					"cpe:/a:redhat:rhel_eus:8.2",
					"cpe:/a:redhat:rhel_eus:8.2::appstream",
					"cpe:/a:redhat:rhel_eus:8.2::crb",
					"cpe:/a:redhat:rhel_eus:8.2::highavailability",
					"cpe:/a:redhat:rhel_eus:8.2::nfv",
					"cpe:/a:redhat:rhel_eus:8.2::realtime",
					"cpe:/a:redhat:rhel_eus:8.2::resilientstorage",
					"cpe:/a:redhat:rhel_eus:8.2::sap",
					"cpe:/a:redhat:rhel_eus:8.2::sap_hana",
					"cpe:/a:redhat:rhel_eus:8.2::supplementary",
					"cpe:/o:redhat:rhel_eus:8.2",
					"cpe:/o:redhat:rhel_eus:8.2::baseos"
				]
			},
			"rights": "Copyright 2022 Red Hat, Inc.",
			"issued": {
				"date": "2022-02-24"
			},
			"bugzilla": [
				{
					"text": "python-pillow: buffer over-read during initialization of ImagePath.Path in path_getbbox() in path.c",
					"href": "https://bugzilla.redhat.com/2042522",
					"id": "2042522"
				},
				{
					"text": "python-pillow: PIL.ImageMath.eval allows evaluation of arbitrary expressions",
					"href": "https://bugzilla.redhat.com/2042527",
					"id": "2042527"
				}
			]
		},
		"affected": {
			"family": "unix",
			"platform": [
				"Red Hat Enterprise Linux 8.2 Extended Update Support"
			]
		}
	},
	"criteria": {
		"operator": "OR",
		"criterias": [
			{
				"operator": "AND",
				"criterias": [
					{
						"operator": "OR",
						"criterions": [
							{
								"test_ref": "oval:com.redhat.rhba:tst:20191992003",
								"comment": "Red Hat Enterprise Linux 8 is installed"
							},
							{
								"test_ref": "oval:com.redhat.rhba:tst:20191992004",
								"comment": "Red Hat CoreOS 4 is installed"
							}
						]
					}
				],
				"criterions": [
					{
						"test_ref": "oval:com.redhat.rhsa:tst:20220667001",
						"comment": "python3-pillow is earlier than 0:5.1.1-14.el8_2"
					},
					{
						"test_ref": "oval:com.redhat.rhsa:tst:20200580002",
						"comment": "python3-pillow is signed with Red Hat redhatrelease2 key"
					}
				]
			}
		],
		"criterions": [
			{
				"test_ref": "oval:com.redhat.rhba:tst:20191992005",
				"comment": "Red Hat Enterprise Linux must be installed"
			}
		]
	}
}
